azkaban-common/src/main/java/azkaban/executor/container/watch/KubernetesWatch.java

```diff
@@ -48,6 +52,12 @@ public class KubernetesWatch implements ContainerizedWatch {
   private final CoreV1Api coreV1Api;
   private final PodWatchParams podWatchParams;
   private final Thread watchRunner;
+  private final int DEFAULT_KUBERNETES_WATCH_CALL_TIMEOUT = 86400;
```

Comment:
Please add a comment stating the units of the timeouts.

Reply:
Or preferably suffix the constant name with "_SECONDS"

Reply:
Also, write the constant values as `days * hours * mins * seconds` to make it more readable, something like `24 * 60 * 60`